lib/linter/linter.js

```diff
@@ -2290,6 +2290,7 @@ class Linter {
         const debugTextDescription = options && options.filename || `${text.slice(0, 10)}...`;
         const shouldFix = options && typeof options.fix !== "undefined" ? options.fix : true;
         const stats = options?.stats;
+        const recentOutputs = [];
```

Comment:
I think the initial text should be in the list so that the autofixing stops if the second pass produces output that is the same as the initial text.